src/app/shared/helpers/config.ts

```diff
@@ -34,4 +34,16 @@ export interface Environment {
 
 type Config = Environment; // config may be someday an extension of the environment
 
-export const appConfig: Config = environment;
+const presetQueryParam = 'config_preset';
+function getPresetNameFromQuery(): string | null {
+  const url = globalThis.location.href;
+  const search = url.includes('?') ? new URLSearchParams(url.slice(url.indexOf('?'))) : null;
+  return search?.get(presetQueryParam) ?? null;
+}
+
+const presetName = getPresetNameByDomain(globalThis.location.hostname) ?? getPresetNameFromQuery();
```

Comment:
You have to explain in code that `getPresetNameFromQuery` in for testing purpose, otherwise each time we'll try to figure out what it is.

Reply:
fixed